tests/DataFixtures/Factory/LocaleFactoryTest.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Tests\DataFixtures\Factory;
+
+use Sylius\Bundle\CoreBundle\DataFixtures\Factory\LocaleFactory;
+use Sylius\Component\Locale\Model\LocaleInterface;
+use Symfony\Bundle\FrameworkBundle\Test\KernelTestCase;
+use Zenstruck\Foundry\Test\Factories;
+use Zenstruck\Foundry\Test\ResetDatabase;
+
+final class LocaleFactoryTest extends KernelTestCase
+{
+    use ResetDatabase;
+    use Factories;
+
+    /** @test */
+    function it_creates_locales(): void
```

Comment:
How about `it_creates_locale_with_random_code`?
Also, all the plural forms seem unnecessary, since it only creates a single instance.